modules/install-consul/install-consul

```diff
@@ -82,11 +82,32 @@ function install_supervisord_amazon_linux {
   sudo pip install supervisor
 
   # On Amazon Linux, /usr/local/bin is not in PATH for the root user, so we add symlinks to /usr/bin, which is in PATH
-  if [[ ! -f "/usr/bin/supervisorctl" ]]; then
-    sudo ln -s /usr/local/bin/supervisorctl /usr/bin/supervisorctl
+  if [[ ! -f "/usr/local/bin/supervisorctl" ]]; then
+    if [[ -f "/usr/bin/supervisorctl" ]]; then
+      sudo ln -s /usr/bin/supervisorctl /usr/local/bin/supervisorctl
+    else
+      echo "Unable to find supervisorctl tool... exiting"
+      exit -5
+    fi
+  elif [[ ! -f "/usr/bin/supervisorctl" ]]; then
+    # IF supervisorctl does NOT exist in /usr/bin, check for it in /usr/local/bin and link it to /usr/bin
+    if [[ -f "/usr/local/bin/supervisorctl" ]]; then
```

Comment:
Should there be an `else` clause here with an error? Same with the if-statements below.

This logic can be tricky to follow, so perhaps extract it into a `two_way_symlink(src, dst)` function that (a) creates a symlink from `src` -> `dst` if `dst` exists and `src` doesn't (b) creates a symlink from `dst` -> `src` if `src` exists and `dst` doesn't, and (c) exits with an error if neither exist.

Reply:
Should be better :)